### PR TITLE
Add hardcoded avatars for some system users.

### LINKED ISSUE #320: give various system accounts their own custom avatar
for instance, the 'bodhi', 'koschei', and 'taskotron' users go around doing all kinds of stuff, but they get the default retro icons for avatars:  https://apps.fedoraproject.org/datagrepper/id?id=2015-1538b186-134f-40cf-af1c-b07d06ed01b6&is_raw=true&size=extra-large

It would be cool if we could have a hardcoded list of special icons for those users.

The bodhi2 server does something like this itself here:  https://github.com/fedora-infra/bodhi/blob/develop/bodhi/util.py#L264-L304

### PR DESCRIPTION
**Pull request by ralphbean**
Fixes #320.

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/fasshim.py [hunk 1]
```diff
@@ -27,7 +27,23 @@ def _ordered_query_params(params):
...
+# https://github.com/fedora-infra/fedmsg_meta_fedora_infrastructure/issues/320
+hardcoded_avatars = {
+    'bodhi': 'https://apps.fedoraproject.org/img/icons/bodhi-{size}.png',
+    'koschei': 'https://apps.fedoraproject.org/img/icons/koschei-{size}.png',
+    # Taskotron may have a new logo at some point.  Check this out:
+    # https://mashaleonova.wordpress.com/2015/08/18/a-logo-for-taskotron/
+    # Ask tflink before actually putting this in place though.  we need
+    # a nice small square version.  It'll look great!
+    # In the meantime, we can use this temporary logo.
+    'taskotron': 'https://apps.fedoraproject.org/img/icons/taskotron-{size}.png'
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/fasshim.py]: Do we have all the sizes?

For example: https://apps.fedoraproject.org/img/icons/taskotron-64.png returns 404 here

Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/fasshim.py]: Ooop!  I missed that one.  I'm uploading it now.

This is what we have currently:

```
roles/apps-fp-o/files/img/icons/bodhi-128.png
roles/apps-fp-o/files/img/icons/bodhi-24.png
roles/apps-fp-o/files/img/icons/bodhi-32.png
roles/apps-fp-o/files/img/icons/bodhi-64.png
roles/apps-fp-o/files/img/icons/koschei-128.png
roles/apps-fp-o/files/img/icons/koschei-24.png
roles/apps-fp-o/files/img/icons/koschei-32.png
roles/apps-fp-o/files/img/icons/koschei-64.png
roles/apps-fp-o/files/img/icons/taskotron-128.png
roles/apps-fp-o/files/img/icons/taskotron-24.png
roles/apps-fp-o/files/img/icons/taskotron-32.png
roles/apps-fp-o/files/img/icons/taskotron-64.png
```

**Comment by ralphbean** [comment]
@pypingou, any further critique on this one?

**Comment by pypingou** [comment]
I have been wondering a little if we should check if the size asked is along the sizes that we support and otherwise return a size we support, but I guess this might introduce un-expected behavior, so might not be the best idea.

:+1: for me

**Comment by ralphbean** [comment]
Thanks Pierre!

**Pull request merged**